default/stringtables/en.txt

```diff
@@ -11594,8 +11600,9 @@ Diversity
 
 PLC_DIVERSITY_DESC
 '''Gives bonuses or penalties to research, stability, and capital influence, dependent on the number of species in the empire.
+The research bonus requires a stability of [[value PLC_DIVERSITY_MIN_STABILITY]].
 
-Gives bigger bonuses or penalities the more above or below 4 species that are present.'''
+All bonuses or penalties are [value PLC_DIVERSITY_SCALING]] per species above or below 4 species that are present.'''
```

Comment:
please double-check the changes...

![diversity](https://user-images.githubusercontent.com/11386908/123556325-e0afac00-d78a-11eb-9084-895a9335fe96.png)

Reply:
I opened all pages, but missed two mistakes anyway. Updated the branched.

Any idea regarding the two points I made in my comment?